libbeat/tests/integration/http_test.go

```diff
@@ -57,12 +57,14 @@ output.console:
 	mockbeat.WriteConfigFile(cfg)
 	mockbeat.Start()
 	mockbeat.WaitForLogs("Starting stats endpoint", 60*time.Second)
+	time.Sleep(time.Second)
```

Comment:
[Blocker]
I see why just `WaitForLogs("Starting stats endpoint"` isn't enough:
https://github.com/elastic/beats/blob/92495664a5674cbde35f51b00970d5db6fabc440/libbeat/api/server.go#L69-L72

but we don't use `time.Sleep` anymore, unless there is no other way. However here there is a better alternative. See below